parsers/verilog.c

```diff
@@ -1031,7 +1037,7 @@ static void createTag (tokenInfo *const token, verilogKind kind)
 		for (unsigned int i = 0; i < ptrArrayCount (tagContents); i++)
 		{
 			tokenInfo *content = ptrArrayItem (tagContents, i);
-			createTag (content, content->kind);
+			createTagWithTypeRef (content, content->kind, NULL);
```

Comment:
I overlooked this yesterday.
First I thought we should add prototype definition of `createTag()` and use `createTag()` as is here.

But after reading the source code, now I agree with your decision.  Using `createTagWithTypeRef()` is better for readability.